modules/administration-guide/partials/proc_configuring_bitbucket_authentication.adoc

```diff
@@ -0,0 +1,52 @@
+// configuring-bitbucket-authentication
+
+[id="configuring_bitbucket_authentication_{context}"]
+= Authentication on Bitbucket servers
+
+{prod} users may use public or private Bitbucket Git repositories as a source of theirs projects. The standard
+factory flow using devfile at the root of the repository is available starting of {version} of {prod}.
+
+Using private repositories, requires some additional configuration described below.
+
+Bitbucket authentication is based on using so-called "personal access tokens". Each Bitbucket user is able to request some
+amount of personal access tokens with different names, permissions, expiration times and so on. Those tokens
+may be used to sign Bitbucket REST API calls and perform Git repository operations.
+
+To allow Bitbucket authentication on {prod} side, personal tokens must be stored in user's namespace in a form of
+secret. The secret must look as follows:
+
+[source,yaml]
+----
+include::example$snip_bitbucket-personal-access-token-secret.adoc[]
+----
+
+The main parts of the secret are:
+
+[cols=2*]
+|===
+| `app.kubernetes.io/component`
+| Label indicates it is a SCM personal token secret
+
+| `che.eclipse.org/che-userid`
+| {prod} id of the user token belongs to
+
+| `che.eclipse.org/scm-userid`
+| SCM id of the user token belongs to
```

Comment:
Is user supposed to create scm userid and username? What if these values are changed and the specified token belongs to another user?

Reply:
rephrased

Reply:
I still don't understand if it's a user/admin who supposed to set userID, expiration time... I'm not familiar with access token on bitbucket but github UI provides me only personal access token (I assume bitbucket does the same, does not provide userid or expiration time for token)